src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/InProcess/SolutionExplorerInProcess.cs

```diff
@@ -142,6 +142,70 @@ public async Task AddFileAsync(string projectName, string fileName, string? cont
         }
     }
 
+    /// <returns>
+    /// The summary line for the build, which generally looks something like this:
+    ///
+    /// <code>
+    /// ========== Build: 1 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========
+    /// </code>
+    /// </returns>
+    public async Task<string> BuildSolutionAndWaitAsync(CancellationToken cancellationToken)
```

Comment:
Taken from the Roslyn tests: https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/SolutionExplorerInProcess.cs#L520.